### ISSUE TITLE
BingX demo trading API

### ISSUE THREAD

**Issue by satekhi** [PipedreamHQ/pipedream#11296]
BingX demo trading API : open-api-vst.bingx.com

**Link to developer documentation**
https://bingx-api.github.io/docs/#/en-us/swapV2/trade-api.html#Trade%20order%20in%20demo%20trading

**Is lack of support preventing you from building workflows, or do you have a workaround?** 

I think at the moment the app for BingX on Pipedearm only connects to real trading API (https://open-api.bingx.com) how can I connect to demo trading API?

**Comment by sergio-eliot-rodriguez** [comment]
This needs a change in the apps config to add a new custom field / prop that allows the user to select the environment they want to
use 
demo - open-api-vst.bingx.com
live - open-api.bingx.com

this app was published outside of the project app store by the team for a customer, the test request is not setup either - likely bcs it uses pre flight signature.

**Comment by satekhi** [comment]
Is there any chance of having that option for selecting an environment though?

**Comment by sergio-eliot-rodriguez** [comment]
Yes, this has been flagged for the team -- that's what i meant for the first part of my previous comment

> 
This needs a change in the apps config to add a new custom field / prop that allows the user to select the environment they want
